package.json

```diff
@@ -113,9 +112,8 @@
     "glob": "^7.1.6",
     "htmlhint": "^0.15.2",
     "ini": "^2.0.0",
-    "mocha": "^8.1.3",
```

Comment:
@YongseopKim According to #328, should I keep `mocha` as it is?

Reply:
Yes. `mocha` will be used. Please do not remove this.